packages/ember-glimmer/tests/integration/helpers/log-test.js

```diff
@@ -0,0 +1,60 @@
+import { RenderingTest, moduleFor } from '../../utils/test-case';
+import { get } from 'ember-metal/property_get';
+import Logger from 'ember-metal/logger';
+
+var originalLog, logCalls;
+
+moduleFor('Helpers test: {{log}}', class extends RenderingTest {
+
+  constructor() {
+    super();
+
+    originalLog = Logger.log;
+    logCalls = [];
+    Logger.log = function(...args) {
+      logCalls.push(...args);
+    };
+  }
+
+  teardown() {
+    Logger.log = originalLog;
+  }
+
+  assertLog(assert, values) {
+    this.assertText('');
+    assert.strictEqual(logCalls.length, values.length);
+
+    for (let i = 0, len = values.length; i < len; i++) {
+      assert.strictEqual(logCalls[i], values[i]);
+    }
+  }
```

Comment:
I think that's generally a good idea. I don't know if we have access to that at that point though. Perhaps setting `this.assert = QUnit.asset` is the same thing? If it literally only takes you a few minutes to investigate and implement it, then go for it. Otherwise let's merge this first and feel free to whip up another PR for that.